source/_posts/2021-03-06-android-2021-q1-releases.markdown

```diff
@@ -0,0 +1,95 @@
+
+## Security Check
+
+As of Home Assistant Core 2021.1.5 some [security vulnerabilities](https://www.home-assistant.io/blog/2021/01/23/security-disclosure2/) were fixed and it is very critical that all users update their instances to at least this version. The app will now do a check every 24 hours to ensure that the instance is at least on the release mentioned in the security alert. You can expect this check to be done anytime a new security alert is issued and the application has been updated.
+
+## Location Disabled Check
+
+We had discovered in several reported issues that certain app features that depend on the connected WiFi network (SSID), requires location to not only be granted to the app but also enabled on the device. Without location the app cannot read the connected SSID which means the Internal URL and any WiFi sensors will stop working. Previously, the application would continue to function and silently fail while showing bad data for the sensors. Although the application still worked, certain parts were found to be buggy as a result of the silent failure.
+
+Some of you may have noticed that starting in 2021.1 the app was showing a prompt before a user was able to interact with the Home Assistant frontend. After 2021.1 we had received a lot of feedback from users about the intrusive pop-up. Starting in 2021.2 this pop-up has been converted to a persistent notification that has its own notification channel allowing the user to fully control how it is displayed on the device. This means you can turn off this notification channel like any other channel on your device. The new channel name for this notification is `Location disabled`. This persistent notification will automatically clear once location is enabled and the application has proper permissions. The notification will list which features are currently disabled.
+
+<p class='img'>
+<img src='/images/blog/2021-03-06-android-q1-releases/location_disabled.png' alt='Screenshot of the location disabled notification'></a>
+Screenshot of the location disabled notification.
+</p>
+
+## High Accuracy Mode
+
+A new feature was added in 2021.2 to allow users to get much faster location updates. This feature will cause additional battery drain. When enabled a persistent notification will be displayed containing some location data. This feature is an enhancement to the Background Location sensor and you can access it from the sensor settings screen. You can also control this feature via a new notification command to enable/disable it on the fly. You can learn more about this feature in the [docs](https://companion.home-assistant.io/docs/core/location#high-accuracy-mode).
```

Comment:
This paragraph is still missing what it exactly does. High accuracy mode now sounds like it just shows a persistent notification 😅 

What does it do exactly?

How is it better than Background update?

```suggestion
A new feature was added in android-2021.2 to allow users to get much faster location updates at the cost of additional battery drain. When enabled, a persistent notification will be displayed containing some location data. This feature is an enhancement to the Background Location sensor and you can access it from the sensor settings screen. You can also control this feature via a new notification command to enable/disable it on the fly. You can learn more about this feature in the [docs](https://companion.home-assistant.io/docs/core/location#high-accuracy-mode).
```

Reply:
Ok good point, I will highlight how much faster it is than the background sensor without it being enabled :)